pkg/kv/kvserver/batcheval/cmd_gc.go

```diff
@@ -184,10 +204,35 @@ func GC(
 		}
 	}
 
+	desc := cArgs.EvalCtx.Desc()
+
+	if rk := args.ClearRangeKey; rk != nil {
+		// Check if we are performing a fast path operation to try to remove all user
+		// key data from the range.
```

Comment:
Point out here that the data must also have been deleted by an MVCC range tombstone.